### DISCUSSION TITLE
Nullable `prismaNode` field

### DISCUSSION THREAD

**Discussion by axhamre** [hayes/pothos#463] [Q&A] [answered]
How do I denote nullability for a `prismaNode` field? For example, the `description` field below. Couldn't find anything in the docs.

<img width="505" alt="CleanShot 2022-07-09 at 20 33 28@2x" src="https://user-images.githubusercontent.com/1907343/178118644-a8b835ee-e451-4eb7-9760-43f667b4a437.png">

**Comment by hayes** [comment, marked as answer]
Just like any other field, `t.exposeString('description', { nullable: true })`

**Reply by axhamre** [reply]
Thank you! 🙏

**Reply by hayes** [reply]
The error message for nullable expose fields is definitely one of the places that Pothos could improve.  Getting the types to be both correct, and still helpful when they fail is a little tricky.
